src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -1010,13 +1038,21 @@ trait SearchConverterService {
               visibleSubjects.flatMap(subject => {
                 val contextFilters = getFilters(topic, subject, bundle, bundle.topicFilterConnections, filterVisibles)
                 val pathIds = (topicPath :+ subject.id).reverse
+                val relevanceId = relevanceIds.head
```

Comment:
Liker ikke helt denne exceptionkastingen heller. Om vi skal fallbacke til kjernestoff så hadde det kanskje gått an å gjort noe sånt?
```suggestion
                val relevanceId = relevanceIds.headOption.getOrElse("urn:relevance:core")
```

Reply:
Hugsa ikkje at headOption var en ting.